operations/iuf/workflows/upgrade_all_products.md

```diff
@@ -1,120 +1,393 @@
+
+    (`ncn-m001#`) Execute the `management-nodes-rollout` stage on all remaining worker and master nodes.
 
     ```bash
-    iuf -a activity-all-products-upgrade -m "${ACTIVITY_DIR}" run --site-vars "${SITE_VARS_FILE}" --bootprep-config-dir "${ACTIVITY_DIR}/hpc-csm-software-recipe-23.05.0/vcs" -r management-nodes-rollout
+    iuf -a "${ACTIVITY_NAME}" run -r management-nodes-rollout
     ```
 
-    Once this step has completed:
+1. Use `kubectl` to remove the `iuf-prevent-rollout=true` label from the canary node.
+
+1. Follow the instructions in [Stage 1 - Ceph image upgrade](#../../upgrade/Stage_1.md) to rebuild the NCN storage nodes.
 
-    - all management NCN nodes have been rebooted to the images and CFS configurations created in previous steps in this example
-    - per-stage product hooks have executed for the `management-nodes-rollout` stage
+1. Follow the instructions in [Stage 2.3 - `ncn-m001` upgrade](#../../upgrade/Stage_2.md#stage-23---ncn-m001-upgrade) to rebuild `ncn-m001`.
 
-1. Invoke `iuf run` with `-r` to execute the [`deploy-product`](../stages/deploy_product.md) and [`post-install-service-check`](../stages/post_install_service_check.md) stages. The `-m`, `--site-vars`, and `--bootprep-config-dir`
-   arguments are the same as the initial invocation of `iuf run` as all steps in this example use the same IUF activity, media, and variables.
+Once this step has completed:
+
+- All management NCN nodes have been rebooted to the images and CFS configurations created in previous steps in this workflow
+- Per-stage product hooks have executed for the `management-nodes-rollout` stage
+
+### Execute the IUF `deploy-product` and `post-install-service-check` stages
+
+1. Refer to the "Install and Upgrade Framework" section of each individual product's installation documentation to determine if any special actions need to be performed outside of IUF for the `deploy-product` or
+`post-install-service-check` stages.
+
+1. Invoke `iuf run` with `-r` to execute the [`deploy-product`](../stages/deploy_product.md) and [`post-install-service-check`](../stages/post_install_service_check.md) stages.
 
     (`ncn-m001#`) Execute the `deploy-product` and `post-install-service-check` stages.
 
     ```bash
-    iuf -a activity-all-products-upgrade -m "${ACTIVITY_DIR}" run --site-vars "${SITE_VARS_FILE}" --bootprep-config-dir "${ACTIVITY_DIR}/hpc-csm-software-recipe-23.05.0/vcs" -r deploy-product post-install-service-check
+    iuf -a "${ACTIVITY_NAME}" run -r deploy-product post-install-service-check
     ```
 
-    Once this step has completed:
+Once this step has completed:
+
+- New versions of product microservices have been deployed
+- Service checks have been run to verify product microservices are executing as expected
+- Per-stage product hooks have executed for the `deploy-product` and `post-install-service-check` stages
+
+### Manual configuration of the Slingshot fabric manager
+
+TBD
```

Comment:
This will typically be for fresh install only unless there has been a system cabling change.

Reply:
Fixed in next update.